cmd/nvidia-vgpu-dm/main.go

```diff
@@ -67,8 +67,6 @@ func main() {
 		return nil
 	}
 
-	log.Infof("version: %s", c.Version)
```

Comment:
This version log is sometimes useful when debugging, why do you wanna get rid of it?

Reply:
In the container image, we always print the version string here: https://github.com/NVIDIA/vgpu-device-manager/blob/main/cmd/nvidia-k8s-vgpu-dm/main.go#L153

However, this CLI, nvidia-vgpu-dm, gets invoked multiple times at runtime when applying vGPU configurations. Therefore, it leads to many log statements repeating the version string.